sonar-project.properties

```diff
@@ -1,18 +1,14 @@
-sonar.projectKey=metamask-extension
-sonar.organization=consensys
+sonar.projectKey=metamask-extension-private
+sonar.organization=metamask
 
-# This is the name and version displayed in the SonarCloud UI.
-sonar.projectName=MetaMask Extension
-#sonar.projectVersion=1.0
+# Source
+sonar.sources=app,development,offscreen,shared,types,ui
+sonar.exclusions=**/*.test.**,**/*.spec.**,app/images
 
-# Root for sonar analysis.
-sonar.sources=app/
+# Tests
+sonar.tests=app,test,development,offscreen,shared,types,ui
+sonar.test.inclusions=**/*.test.**,**/*.spec.**
+sonar.javascript.lcov.reportPaths=tests/coverage/lcov.info
 
-# Excluded project files from analysis.
-#sonar.exclusions=
-
-# Inclusions for test files.
-sonar.test.inclusions=**.test.**
-
-# Encoding of the source code. Default is default system encoding
-sonar.sourceEncoding=UTF-8
+# Fail CI job if quality gate failures
+sonar.qualitygate.wait=false
```

Comment:
More of a question than a change request, do we really want to see the job green, even if it fails? 
We are not requiring this job to pass, so we can still display it as failed, but not block the ci

https://github.com/MetaMask/metamask-extension/blob/develop/.circleci/config.yml#L276

Reply:
changing my thought, I think it's fine as it is, as we don't want to see red jobs even if non-blocking, unless until this has been stabilized :+1:

Reply:
yeah, we are not yet there